ga4gh/datamodel/reads.py

```diff
@@ -242,7 +256,7 @@ def _referenceSetInit(self, dataRepository, shouldThrowExceptions):
             elif referenceSetName != name:
                 if shouldThrowExceptions:
                     raise exceptions.MultipleReferenceSetsInReadGroupSet(
-                        self._samFilePath, name, referenceSetName)
+                        self._dataUrl, name, referenceSetName)
```

Comment:
We definitely want to make this more robust. It would be nice if we could detect the reference set by looking at the md5s at import time automatically (it would save the user some effort), but this should probably be pushed up the stack closer to the repo manager UI.